azkaban_cli/azkaban_cli.py

```diff
@@ -566,6 +596,16 @@ def fetch_flow_execution(ctx, execution_id):
     """Fetch a flow execution"""
     __fetch_flow_execution(ctx, execution_id)
 
+@click.command(u'fetch_executions_of_a_flow')
+@click.pass_context
+@click.argument(u'project', type=click.STRING)
+@click.argument(u'flow', type=click.STRING)
+@click.argument(u'start', type=click.INTEGER)
+@click.argument(u'length', type=click.INTEGER)
```

Comment:
I would _maybe_ have default parameters here for `start` and `length`, like start=0 and length=3? Just so that there's a default behavior really.
@ggalves Do you agree or is it fine like this? The PR looks good to me

Reply:
> I would _maybe_ have default parameters here for `start` and `length`, like start=0 and length=3? Just so that there's a default behavior really.
> @ggalves Do you agree or is it fine like this? Otherwise the PR looks good to me

I thought about it, it sounds very reasonable, but i didnt know which values would fit your purpose.

Reply:
I'd say 0 and 3 for defaults should be fine. Maybe 0 and 5. 0 and anything really, since it's just a default value 😂

Reply:
i've added the default values to the cli command, i think that does the trick (: